xpdacq/beamtimeSetup.py

```diff
@@ -220,16 +220,15 @@ def _end_beamtime(base_dir=None, archive_dir=None, bto=None, usr_confirm='y'):
     # laod bt yaml
     ips = get_ipython()
     if not bto:
-        # bto = _load_bt(glbl.yaml_dir)
         bto = ips.ns_table['user_global']['bt']
     # load bt info
     archive_name = _load_bt_info(bto, _required_info)
     # archive file
-    archive_full_name = _tar_user_data(archive_name)
+    archive_full_name, local_archive_name = _tar_user_data(archive_name)
     # confirm archive
     _confirm_archive(archive_full_name)
     # flush
-    _delete_home_dir_tree()
+    _delete_archive_dir_tree(local_archive_name)
```

Comment:
this function name is confusing.  It looks as if you are deleting the archive.

maybe change it to ``_delete_local_archive``?

Reply:
yeah, agree. I will update the name.